tilequeue/command.py

```diff
@@ -1423,77 +1410,59 @@ def tilequeue_dump_tiles_of_interest(cfg, peripherals):
     )
 
 
-def tilequeue_load_tiles_of_interest(cfg, peripherals):
-    """
-    Given a newline-delimited file containing tile coordinates in
-    `zoom/column/row` format, load those tiles into a new Redis SET
-    alongside the existing one, then RENAME the new to the old.
-    """
-    logger = make_logger(cfg, 'load_tiles_of_interest')
-    logger.info('Loading tiles of interest')
+def tilequeue_dump_tiles_of_interest_from_redis(cfg, peripherals):
```

Comment:
Right. I'll probably put together a PR after this one to remove this bit of code altogether.